libs/output-mapping/src/Writer/Helper/ConfigurationMerger.php

```diff
@@ -37,6 +37,10 @@ public static function mergeConfigurations(array $configFromManifest, array $con
                 if (!isset($config[$key]) || is_array($value)) {
                     $config[$key] = $value;
                 }
+            } elseif ($key === 'table_metadata') {
```

Comment:
K čemu je to dobrý přepisovat metadata? Nemělo by tu být jen schema?

Reply:
`table_metadata` je nový nastavení místo `metadata`a je to key => value pole tak jsem to sem taky přidal aby se dalo mergovat

Reply:
💡 aha, prehlidl jsem ze nahore se `metadata` uz merguji, to jsem ani nevedel :)